doc/samples/sm_ppp_shell.rst

```diff
@@ -0,0 +1,254 @@
+* CMUX (GSM 07.10) multiplexing support
+* Power management with runtime power saving
+
+For more information about using the nRF91 Series as a Zephyr-compatible modem, see :ref:`sm_as_zephyr_modem`.
+
+Requirements
+************
+
+The |SM| application must be built with the following overlays:
+
+* :file:`app/overlay-cmux.conf` - To enable CMUX support
+* :file:`app/overlay-ppp.conf` - To enable PPP support
+
+If using an external MCU as the controlling device, also include:
+
+* :file:`app/overlay-external-mcu.overlay` - To configure UART pins, DTR, and RI signals
+
+The sample supports the following development kits:
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Hardware platforms
+     - PCA
+     - Board name
+     - Board target
+   * - `nRF54L15 DK <nRF54L15 DK_>`_
+     - PCA10156
+     - `nrf54l15dk`_
+     - ``nrf54l15dk/nrf54l15/cpuapp``
+   * - `nRF54L15 DK <nRF54L15 DK_>`_ (TrustZone)
+     - PCA10156
+     - `nrf54l15dk`_
+     - ``nrf54l15dk/nrf54l15/cpuapp/ns``
+   * - Native simulator
+     - N/A
+     - ``native_sim``
+     - ``native_sim``
+
+Hardware setup for nRF54L15 DK
+===============================
+
+When using the nRF54L15 DK as the controlling device, connect the DK pins to the corresponding pins on the nRF91 Series DK as follows:
+
+.. list-table::
+   :header-rows: 1
+
+   * - nRF54L15 DK
+     - nRF91 Series DK
+   * - UART TX P0.00
+     - UART RX P0.11
+   * - UART RX P0.01
+     - UART TX P0.10
+   * - UART CTS P0.03
+     - UART RTS P0.12
+   * - UART RTS P0.02
+     - UART CTS P0.13
+   * - DTR OUT P1.11
+     - DTR IN P0.31
+   * - RI IN P1.12
+     - RI OUT P0.30
+   * - GPIO GND
+     - GPIO GND
+
+.. note::
+   You must disable the VCOM0 on the nRF54L15 DK to release the UART GPIO pins for communication with the Serial Modem.
+
+   * Use the `Board Configurator app`_ to disable the ``Connect port VCOM0`` setting.
+
+.. note::
+   The GPIO output levels on the nRF91 Series device and nRF54L15 DK must be the same.
+
+   * Set the VDD voltages for both devices using the `Board Configurator app`_.
+
+Native simulator setup
+======================
+
+When running on the native simulator, the sample expects the |SM| to be available at ``/dev/ttyACM0`` on the Linux host system.
+Connect the nRF91 Series DK running |SM| to your Linux host via USB.
+
+Building and running
+********************
+
+Building for nRF54L15 DK
+=========================
+
+To build the sample for the nRF54L15 DK:
+
+.. code-block:: bash
+
+   west build -b nrf54l15dk/nrf54l15/cpuapp
+
+For the TrustZone-enabled variant:
+
+.. code-block:: bash
+
+   west build -b nrf54l15dk/nrf54l15/cpuapp/ns
+
+Building for native simulator
+==============================
+
+To build the sample for the native simulator:
+
+.. code-block:: bash
+
+   west build -b native_sim
+
+Running the sample
+==================
+
+Flash the sample to your board and open a serial terminal to view the shell interface.
```

Comment:
This is also for nRF54, right?
I would include the running part in the build sections above for specific targets and modify those sections from "Building for nRF54L15 DK" to just "nRF54L15 DK". With running instructions included, this would match the higher level caption of "Building and running".